lib/Echidna/Campaign.hs

```diff
@@ -201,6 +232,8 @@ callseq v w ql = do
       diffs = H.fromList [(AbiAddressType, S.fromList $ AbiAddress . addressWord160 <$> diff)]
   -- Save the global campaign state (also vm state, but that gets reset before it's used)
   hasLens .= snd s
+  when (s ^. _2 . newCoverage) $ addToCorpus res
+  hasLens . ncallseqs %= (+1)  
```

Comment:
pretty sure there is `(+=)`

Reply:
It's the same. I changed anyway, since that could be easier to read.